### ISSUE TITLE
Crates compat with Create moving contraptions

### ISSUE THREAD

**Issue by joseth257** [MysticMods/Druidcraft#91]
### General Information

**Describe the bug:**
Druidcraft crates are attached to Create contraptions successfully, but breaking/stopping said contraption ends on a FATAL error. The game does not crash, but the crate, and some of the blocks it is attached to, disappears. 
I have not found any other modded chest like block triggering such error on create so far. So this should probably be related to druidcraft's crates complicated multiblock behavior. 
 
**To Reproduce:**
1. Install Create + Druidcraft
2. Create a moving contraption (example: cart assembler + linear chassis)
3. Attached a crate to it using slime
4. Run a cart through it
5. Stop the contraption and break the minecart

**Expected behavior:**
As the contraption stops, the crate should be recreated on it's "static" form (with its possible changed inventory due to create fancy interactions).

***

### Environment Versions
Druidcraft-1.16.5-0.4.52
create-mc1.16.3_v0.3e

**Mystic Mods Versions**
- None

**Other Versions:**
- Conflicting mod (if regarding mod integration): Create
- Other mods you think could cause issues: Create
- Forge: 36.0.1
- Minecraft: 1.16.5
- Modpack (if available): Custom modpack with [ Druidcraft + Create ] on Multimc (Debian)

***

### Logging Information
**Please Use [gist](https://gist.github.com), [paste.ee](https://paste.ee), [pastebin](https://pastebin.com) or some other log sharing service**

**Crash Report (if available):**

**Latest Log (if necessary):**

https://pastebin.com/D83LFiMV
***

### Additional Information
**Screenshots (if available):**
![Screenshot from 2021-01-25 20-44-31](https://user-images.githubusercontent.com/64246231/105779988-48e53200-5f4e-11eb-9269-fbbccce2da53.png)
![Screenshot from 2021-01-25 20-44-43](https://user-images.githubusercontent.com/64246231/105780105-86e25600-5f4e-11eb-9fff-f9fc8596fe9b.png)

**Additional context (optional):**

I have seen others block with inventory not being able to be attached to create chassis, so at worst if it comes down to an unsolvable conflict due to the crate's multiblock fanciness, the crate should at least not be an acceptable piece on the contraption (example: mekanism personal chest or bin).

**Comment by LordGrimmauld** [comment]
Yes i am aware. This has been a bug since Create 0.2.4 since we increased the compat for inventories. A (temporary) fix for you would be tagging crates as `create:immovable` (block tag). That is until we either implement the tag from Create side, Druidcraft implements that tag, or we get an actual fix in code done. Can confirm, this indeed happens and is 100% reproduceable.

**Comment by noobanidus** [comment]
@LordGrimmauld Thanks for the update on this, I'll look into doing that. Although the crash that it does cause *shouldn't* happen from our end because it's assuming that the blockstate is the crate blockstate when it isn't, and that's invalid logic. That's something I'd like to fix separately.

**Comment by Tander** [comment]
@noobanidus Hi. It's not exactly a duplicate, I don't even have Create in my pack, it's just Druidcraft's crate and vanilla pistons. But I guess underlying problem is the same, so, ok.
